stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/attachments/content/FileUploadContent.kt

```diff
@@ -52,10 +57,21 @@ public fun FileUploadContent(
     modifier: Modifier = Modifier,
 ) {
     val message = attachmentState.message
+    val previewHandlers = ChatTheme.attachmentPreviewHandlers
 
     Column(modifier = modifier) {
         for (attachment in message.attachments) {
-            FileUploadItem(attachment = attachment)
+            FileUploadItem(
```

Comment:
Same here, a modifier should be fine to pass down and abstract away the click handlers

Reply:
Passed in a modifier that's styled at the call site.

That way the Composable isn't "dirtied" with a pre set modifier that has `.then()` on it.